### PR TITLE
Related component 

### PR DESCRIPTION
**Pull request by nickhsine**
### Reference
[KanbanFlow](https://kanbanflow.com/t/UQ5vnUs2)

### CODE REVIEW

File: src/components/related/card.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        year: 'numeric',
+        month: 'numeric',
+        day: 'numeric',
+      })
+    : ''
+
+  return (
+    <Block>
+      <Thumbnail>
+        <img src={props.thumbnail} />
```
Comment (YuCJ) [inline src/components/related/card.js]: TODO: We can use `components/shared/img-with-placeholder` to replace `img` after the style is confirmed

File: src/components/related/index.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  border-right: solid 0.5px #d8d8d8;
+`
+
+const UpperSeparationLine = styled(SeparationLine)`
+  border-top: solid 0.5px #d8d8d8;
+  margin-bottom: -2px;
+`
+
+const List = styled.div`
+  display: inline-flex;
```
Comment (YuCJ) [inline src/components/related/index.js]: Why we choose `inline-flex` rather than `flex`?

**Review by YuCJ — COMMENTED** [review]
LGTM 👍

File: src/components/related/card.js (hunk 1 above)
Comment (nickhsine) [reply src/components/related/card.js]: Would use `img-with-placeholder` after this PR merged(https://github.com/twreporter/react-article-components/pull/10)

**Comment by nickhsine** [comment]
Updated according new mockup.

File: src/components/related/card.js [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+      display: block;
+      position: absolute;
+      z-index: 1;
+      font-size: 14px;
+      line-height: 1.43;
+      color: ${props => props.theme.colors.base.lightText};
+      background-color: #f1f1f1;
+      padding: 15px;
+      border-radius: 4px;
+      margin-top: 12px 0;
```
Comment (YuCJ) [inline src/components/related/card.js]: `margin-top: 12px;`

**Review by YuCJ — COMMENTED** [review]
LGTM 👍

**Pull request merged**